overlord/devicestate/devicemgr.go

```diff
@@ -554,6 +554,36 @@ func (m *DeviceManager) ensureInstalled() error {
 	return nil
 }
 
+var timeNow = time.Now
+
+// OperationalTime returns the time when snapd became operational, and sets it
```

Comment:
"become operational" has a historical meaning of having registered, which is not what we mean here.  So we need a different name.  Maybe just 

"OperatingTime returns the time when snapd started operating. ..."

Otherwise we should use the longer  StartOfOperationTime (start-of-operation-time) ?

Reply:
Right, I feeled this name could be confusing... StartOfOperationTime sounds good and works for me, thanks.